libbindgen/tests/headers/disable-namespacing.hpp

```diff
@@ -0,0 +1,9 @@
+// bindgen-flags: --disable-name-namespacing
+
+namespace foo {
+namespace bar {
+
+typedef int Baz;
```

Comment:
> I think this is fair to do in this case, given we warn properly.

Sure. We just need to document it really well. Ideally, we would also test this behavior, but I don't think our test suite can handle expected failures...